src/main/java/org/opensrp/web/rest/PractitionerResource.java

```diff
@@ -142,7 +140,14 @@ public void setPractitionerService(PractitionerService practitionerService) {
         }
     }
 
-    private PractitionerSearchBean createPractitionerSearchBean(Integer pageNumber, Integer pageSize, String orderByType,
+    @GetMapping(value = "/report-to", produces = { MediaType.APPLICATION_JSON_VALUE })
+    public List<Practitioner> getPractitionersByPractitionerRoleIdentifierAndCode(@RequestParam(value = "practitionerIdentifier") String practitionerIdentifier,
+            @RequestParam(value = "code") String code) {
+
+        return practitionerService.getAssignedPractitionersByIdentifierAndCode(practitionerIdentifier, code);
+    }
+
+	private PractitionerSearchBean createPractitionerSearchBean(Integer pageNumber, Integer pageSize, String orderByType,
```

Comment:
Identation is different. This is a quick fix

Reply:
Updated